src/instructlab/utils.py

```diff
@@ -286,15 +299,45 @@ def _validate_documents(
             opened_files = False
             for pattern in file_patterns:
                 for file_path in glob.glob(os.path.join(repo.working_dir, pattern)):
-                    if os.path.isfile(file_path) and file_path.endswith(".md"):
-                        with open(file_path, "r", encoding="utf-8"):
-                            # Success! we could open the file.
-                            # We don't actually care about the contents, though.
-                            opened_files = True
+                    logger.debug("Checking file: %s", file_path)
+                    if os.path.isfile(file_path):
+                        file_extension = os.path.splitext(file_path)[1]
+                        file_info = file_types.get(file_extension)
```

Comment:
ok so, you might need to assert the type of this before using it. Either that, or assert the type of the `file_types` dict above before instantiating it. 

you can assert the type of `file_info` here by doing `assert` or `isinstance`

Reply:
this is the only issue I think

Reply:
fixed